blog/_posts/2018-04-18-scala-3.md

```diff
@@ -0,0 +1,104 @@
+---
+layout: blog-detail
+post-type: blog
+by: Martin Odersky
+title: "Towards Scala 3"
+---
+
+Now that Scala 2.13 is around the corner, it's time to consider the
+roadmap beyond it. It's been no secret that the work on Dotty over the
+last 5 years was intended to explore what a new Scala could look
+like. We are now at a stage where we can commit: Dotty will become
+Scala 3.0.
+
+Of course, this statement invites many follow-up questions. Here are
+some answers we can already give today. We expect there will be more
+questions and answers as things shape up.
+
```

Comment:
I have repeatedly encountered a persistent misconception among new and/or occasional Scala users that Dotty is somehow a new language. I suggest leading with a "_How different are Scala 2 and Scala 3?_" section that emphasizes continuity. Perhaps beginning with something like "Scala 2 and Scala 3 are fundamentally the same language. The compiler is new, but nearly everything Scala programmers already know about Scala 2 applies to Scala 3 as well, and most ordinary Scala 2 code will also work on Scala 3 with only minor changes."

Reply:
Good idea, I have added that wording.